src/lib.rs

```diff
@@ -185,7 +175,7 @@ where
 }
 
 #[inline]
-fn get_size_of_rows(m: usize) -> usize {
+fn get_depth(m: usize) -> usize {
```

Comment:
https://www.quora.com/What-is-the-difference-between-height-and-depth-of-a-tree

> Height is measured in upward direction. The longest path from leaf to the root is called height of a tree.
> 
> Depth is always measured in downward direction. The length of the path from root to the end is called depth of the tree.

> The depth of a node is the number of edges from the node to the tree's root node.
> A root node will have a depth of 0.
> The height of a node is the number of edges on the longest path from the node to a leaf. A leaf node will have a height of 0.
> Height and depth of a tree is equal but height and depth of a node is not equal because the height is calculated by traversing from the given node to the deepest possible leaf. Depth is calculated from traversal from root to the given node.
> 
> For more info get help at Computer Science Assignment Help

Reply:
```
    #[test]
    fn test_depth() {
        let check = vec![
            (0, 1),
            (1, 1),
            (2, 2),
            (3, 3),
            (4, 3),
            (5, 4),
            (8, 4),
            (9, 5),
            (16, 5),
            (17, 6),
        ];
        for (x, y) in check {
            assert_eq!(y, super::get_depth(x));
        }
    }
```

I think the test is not correct.